src/MongoDB/Command.c

```diff
@@ -69,6 +72,19 @@ static bool php_phongo_command_init(php_phongo_command_t *intern, zval *filter,
 		return false;
 	}
 
+	if (bson_iter_init (&iter, intern->bson) && bson_iter_find_descendant (&iter, "cursor.batchSize", &sub_iter) && BSON_ITER_HOLDS_INT(&sub_iter)) {
```

Comment:
Spaces between the function name and open parenthesis is part of libmongoc's coding standard, but we don't follow it within the PHP extension. `bson_iter_init(...)` works fine.